packages/veritone-react-common/src/components/MediaDetails/index.js

```diff
@@ -12,13 +13,23 @@ export default class MediaDetails extends Component {
   };
 
   state = {
-    selectedEngineCategoryId: ''
+    selectedEngineCategory: this.props.engineCategories && this.props.engineCategories.length ? this.props.engineCategories[0] : null
```

Comment:
Is this something we can move to redux?

Reply:
yes, and it would feed it down the stack I think